app/src/main/java/com/ncnf/utilities/map/MapHandler.java

```diff
@@ -50,14 +51,13 @@
      * and handling the retrieval of the events and organizations to be displayed
      * @param context The activity the map will be displayed in
      * @param mMap The Google Map to display
-     * @param venueProvider A provider for the organizers, or venues.
      * @param fragmentManager A children fragment manager from the MapFragment
-     * @param databaseService the service to query the database
+     * @param eventRepository the service to query the database
      */
-    public MapHandler(AppCompatActivity context, GoogleMap mMap, FragmentManager fragmentManager, DatabaseService databaseService){
+    public MapHandler(AppCompatActivity context, GoogleMap mMap, FragmentManager fragmentManager, EventRepository eventRepository){
```

Comment:
This will probably clash with @Arthelh current PR. Need to decide who merges first.

Reply:
I think it will yes. If that's okay with you Jeremie I'll merge first and you'll just have to add an OrganizationRepository in addition to the EventRepository as I also use the databaseService for Organizations now

Reply:
I'll handle the merging if there is issues, but @Arthelh knows what I did on this part !